### ISSUE TITLE
PLSQL %type and %rowtype

### ISSUE THREAD

**Issue by glgraca** [antlr/grammars-v4#1414]
I ran tests on my packages and found that I needed to make some changes to the lexer:

```antlr
PERCENT_FOUND:                '%' SPACE* 'FOUND';
PERCENT_ISOPEN:               '%' SPACE* 'ISOPEN';
PERCENT_NOTFOUND:             '%' SPACE* 'NOTFOUND';
PERCENT_ROWCOUNT:             '%' SPACE* 'ROWCOUNT';
PERCENT_ROWTYPE:              '%' SPACE* 'ROWTYPE';
PERCENT_TYPE:                 '%' SPACE* 'TYPE';
```

**Comment by teverett** [comment]
Please submit a PR.

**Comment by KvanTTT** [comment]
Fixed by #1415
